### PR TITLE
Document the log manipulation facilities.

### PR DESCRIPTION
**Pull request by lmlg**

### CODE REVIEW

File: docs/how-to/change-log-level.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+By default, the microceph daemon runs with the log level set to DEBUG. While that is the desirable
+behaviour for a good amount of use cases, there are instances when this level is far too high -
+for example, embedded devices where storage is much more limited. For these reasons, the microceph
+daemon exposes a way to both get and set the log level.
+
+Configuring the log level
+-------------------------
+
+Microceph includes the command `log`, with the sub commands `set-level` and `get-level`. When setting, we support both string and integer formats for the log level. For example:
```
Comment (sabaini) [inline docs/how-to/change-log-level.rst:13]: Nit: s/Microceph/MicroCeph/

File: docs/how-to/change-log-level.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
+===============================
+Changing log level in microceph
```
Comment (sabaini) [inline docs/how-to/change-log-level.rst:2]: Nit: s/microceph/MicroCeph/

**Review by sabaini — CHANGES_REQUESTED** [review]
Thanks! Two minor branding nits / suggestions

File: docs/how-to/change-log-level.rst (hunk 2 above)
Comment (UtkarshBhatthere) [reply docs/how-to/change-log-level.rst:2]: suggestion: s/Changing/change so that it fits better in the `How to "Change log level in MicroCeph"` format.

**Review by UtkarshBhatthere — CHANGES_REQUESTED** [review]
1 minor nit

File: docs/how-to/change-log-level.rst (hunk 2 above)
Comment (lmlg) [reply docs/how-to/change-log-level.rst]: done.

File: docs/how-to/change-log-level.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
+===============================
+Change log level in MicroCeph
+===============================
```
Comment (pmatulis) [inline docs/how-to/change-log-level.rst:3]: let's just say: "Changing the log level"
and ensure that the underlines/overlines are the same length

File: docs/how-to/change-log-level.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+By default, the MicroCeph daemon runs with the log level set to DEBUG. While that is the desirable
+behaviour for a good amount of use cases, there are instances when this level is far too high -
```
Comment (pmatulis) [inline docs/how-to/change-log-level.rst:6]: s/amount/number

File: docs/how-to/change-log-level.rst [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+for example, embedded devices where storage is much more limited. For these reasons, the MicroCeph
+daemon exposes a way to both get and set the log level.
+
+Configuring the log level
+-------------------------
+
+MicroCeph includes the command `log`, with the sub commands `set-level` and `get-level`. When setting, we support both string and integer formats for the log level. For example:
```
Comment (pmatulis) [inline docs/how-to/change-log-level.rst:13]: * double backticks for monospace (commands). single ones give italics in RST
* s/sub command/sub-command

(change everywhere in the document)

File: docs/how-to/change-log-level.rst [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+   sudo microceph log set-level 3
+
+Both commands are equivalent. The mapping from integer to string can be consulted by querying the
+help for the `set-level` sub command. Note that any changes made to the log level take effect
+immediately, and need no restarts.
+
+On the other hand, the `get-level` sub command takes no arguments and returns an integer level only.
+Any value returned by `get-level` can be used for `set-level`.
+
+For example, after setting the level as shown in the example, we can run and verify the following:
```
Comment (pmatulis) [inline docs/how-to/change-log-level.rst:27]: ...we can verify in this way:

**Review by pmatulis — CHANGES_REQUESTED** [review]

File: docs/how-to/change-log-level.rst (hunk 3 above)
Comment (lmlg) [reply docs/how-to/change-log-level.rst:3]: done.

File: docs/how-to/change-log-level.rst (hunk 5 above)
Comment (lmlg) [reply docs/how-to/change-log-level.rst]: done.

File: docs/how-to/change-log-level.rst (hunk 4 above)
Comment (lmlg) [reply docs/how-to/change-log-level.rst]: done.

File: docs/how-to/change-log-level.rst (hunk 3 above)
Comment (pmatulis) [reply docs/how-to/change-log-level.rst:3]: just "Changing the log level"

as in all other page titles, it's understood to be MicroCeph

**Review by pmatulis — CHANGES_REQUESTED** [review]

**Review by UtkarshBhatthere — APPROVED** [review]
LGTM

**Pull request merged**